crt_portal/cts_forms/templates/forms/complaint_view/show/index.html

```diff
@@ -2,52 +2,52 @@
 {% load static %}
 
 {% block content %}
-  <div class="complaint-show-body">
-    <div class="grid-container-widescreen">
-      <div class="grid-row margin-bottom-1">
-        <div class="tablet:grid-col-4 grid-offset-1 padding-left-05">
-          <div class="display-flex">
-            <a class="outline-button outline-button--dark" href="/form/view{{ return_url_args }}">
-              <img src="{% static "img/intake-icons/ic_arrow_forward.svg" %}" class="icon">
-              Back to all
-            </a>
-          </div>
-          <div class="display-flex flex-align-center details-id">
-            <h2 class="margin-right-205 margin-top-0 margin-bottom-0 backend-blue">
-              ID: {{ data.public_id }}
-            </h2>
-            <span class="status-tag status-{{data.status}} margin-top-05">
-              {{ data.status }}
-            </span>
-          </div>
-          <p>
-            <span class="details-date-label text-uppercase backend-blue">Received: </span>
-            <span class="backend-blue">{{ data.create_date|date:'g:i a F j, Y' }}</span>
-          </p>
-          <p>
-            <span class="details-date-label text-uppercase backend-blue">Last Updated: </span>
-            <span class="backend-blue">{{ data.modified_date|date:'g:i a F j, Y' }}</span>
-          </p>
+<div class="complaint-show-body">
```

Comment:
Augh, it looks like my editor also changed a bunch of HTML from 4 spaces to 2. I'll fix this on my side. Ignore for now.

Reply:
I think that is right, It was probably an issue on my end- No worries!